gcp/api/server.py

```diff
@@ -626,6 +621,10 @@ def do_query(query, context: QueryContext, include_details=True):
     purl_str = ''
 
   if ecosystem and not ecosystems.get(ecosystem):
+    if ecosystem.startswith('Alpine') and not ecosystem.startswith('Alpine:v'):
```

Comment:
I'll move the Alpine release number check to a followup PR since it will affect both test and prod instances.